### ISSUE TITLE
Ubuntu 18 UWSGI

### ISSUE THREAD

**Issue by itdependsnetworks** [nautobot/nautobot#116]
Opening an issue to immediately close for documentation purposes.

Per user on Slack. 

If you are running Ubuntu 18.x you might run into an issue with  uwsgi.ini listen set to 1024. Ubuntu 18.x by default is set to 128. You can check that and change it via the following commands (note: you have to be root to do this) `cat /proc/sys/net/core/somaxconn`  if it says 128 then you can change it by issues the following command -  `echo 1024 > /proc/sys/net/core/somaxconn`

**Comment by jathanism** [comment]
This is helpful information but we've only provided official documented support for Ubuntu 20.04 at this time since that's what we inherited from when we forked NetBox. 

Before I'd be confident supporting older versions of Ubuntu, I'd want to assert whether they or not they are LTS and then certify them.

Another important point, though, that we should clarify is that the existing uWSGI settings we just recently added are not necessarily prescriptive. They are derived from what we have been using for customer environments, but doesn't necessarily mean that will be what works for everyone. uWSGI has a dizzying amount of configuration options that will be hard to get right.
